chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/urlStatus/QuestionableUrlReportGenerator.java

```diff
@@ -282,19 +282,8 @@ private File getOutputFile(List<FailedUrlResult> urlResultsToWrite, String repor
                 // urlResultsToWrite must be sorted by url
                 for (int i = 0; i < urlResultsToWrite.size(); i++) {
                     FailedUrlResult currUrlResult = urlResultsToWrite.get(i);
-                    FailedUrlResult prevUrlResult = null;
-                    if (i > 0) {
-                        prevUrlResult = urlResultsToWrite.get(i - 1);
-                    }
                     List<String> rowValue = null;
-                    if (prevUrlResult == null || !currUrlResult.getUrl().equals(prevUrlResult.getUrl())) {
-                        // write a row with the url data since this url is different than the one before it
-                        rowValue = generateRowValue(currUrlResult, true);
-                    } else {
-                        // write a row with just the acb/atl/developer/listing data since this is for the same url
-                        // as the one before it
-                        rowValue = generateRowValue(currUrlResult, false);
-                    }
+                        rowValue = generateRowValue(currUrlResult);
```

Comment:
This indentation looks incorrect. Do you have checkstyle configured and turned on? It should alert you about style issues like this